tests/extensions/edm/utils.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+EDM utils
+-------------
+"""
+import json
+
+# from tests import utils as test_utils
+# from flask import current_app
+# import os
+# import shutil
+# from app.extensions.tus import tus_upload_dir
+
+CONFIG_PATH = '/api/v1/configuration/default'
+
+
+def configuration_post(
+    flask_app_client,
+    user,
+    data_in={},
+    expected_status_code=200,
+):
+    with flask_app_client.login(user, auth_scopes=('configuration:write',)):
+        response = flask_app_client.post(
+            '%s' % CONFIG_PATH,
+            data=json.dumps(data_in),
+            content_type='application/json',
+        )
+    assert response.status_code == expected_status_code
+    assert isinstance(response.json, dict)
```

Comment:
any reason you're not using the test_utils.validate_dict_response?

Reply:
not really; just kinda rolled this by hand real quick.

Reply:
i also kinda imagined this _generic_ configuration POST thing being used elsewhere (despite it only being used once so far) ... so was guessing other cases would be checked here.  guess i could still use this helper function.